views/components/ConfigurationsList/index.jsx

```diff
@@ -715,36 +365,40 @@ const ConfigurationsList = ({ user, classes, theme }) => {
     }),
   }
 
-  if (state?.configurations?.length <= 0) return <div>Loading...</div>
+  if (configurations?.length <= 0) return <div>Loading...</div>
```

Comment:
```suggestion
  if (configurations.length === 0) return <div>Loading...</div>
```

Reply:
[d31c716](https://github.com/AMP-SCZ/dpdash/pull/310/commits/d31c716b584e340b90068954fc940c49761e52fe)

Reply:
this comment still stands. The safe navigator can be removed, and it's not possible for the length of the array to be less than 0.

Reply:
[22063e9](https://github.com/AMP-SCZ/dpdash/pull/310/commits/22063e9b9ef9c557741703dee541a7e18fe68e26)